slickqaweb/static/resources/shared/main/main.js

```diff
@@ -197,15 +308,86 @@ angular.module('slickApp')
+        $scope.healthDataByProject = {};
+
+        let params = {limit: $scope.buildsQuery.queryLimit, groupType: "PARALLEL"};
+        if ($routeParams["limit"]) {
+            params.limit = $routeParams["limit"];
+        }
+
+        let healthIntervals = {};
+
+        $scope.getHealthData = function (project, release) {
+            rest.one('release-report', project).one(release).get(params).then(function (releaseReport) {
+                $scope.healthReportOptionsByProject[project] = {
+                    chartArea: {left: '5%', top: '5%', width: '85%', height: '80%'},
+                    backgroundColor: "none",
+                    vAxis: {
+                        minValue: 0,
+                        maxValue: 100,
+                        format: '#\'%\''
+                    },
+                    lineWidth: 5,
+                    legend: {
+                        textStyle: {
+                            color: "#ffffff"
+                        }
+                    },
+                    colors: []
+                };
+                if (releaseReport.hasOwnProperty('name')) {
+                    $scope.healthDataByProject[project] = new google.visualization.DataTable();
+                    $scope.healthDataByProject[project].addColumn('date', 'Recorded');
+                    let gotXAndY = false;
+                    _.each(releaseReport.builds.sort(function (a, b) {
+                        return (a.testruns[0].dateCreated > b.testruns[0].dateCreated) ? 1 : ((b.testruns[0].dateCreated > a.testruns[0].dateCreated) ? -1 : 0);
+                    }), function (build) {
+                        let row = [new Date(build.testruns[0].dateCreated)];
+                        let sum = Object.values(build.groupSummary.resultsByStatus).reduce((a, b) => a + b, 0);
+                        if (!gotXAndY) {
+                            _.each(Object.keys(build.groupSummary.resultsByStatus).sort(), function (status) {
+                                let color = getStyle(replaceOnStatus(status, "") + "-element", "color");
+                                $scope.healthReportOptionsByProject[project].colors.push(color);
+                                $scope.healthDataByProject[project].addColumn('number', replaceOnStatus(status, " "));
+                            });
+                            gotXAndY = true;
+                            //$scope.healthDataByProject[project].addColumn('string', 'Build');
+                        }
+                        _.each(Object.keys(build.groupSummary.resultsByStatus).sort(), function (status) {
+                            row.push(build.groupSummary.resultsByStatus[status] / sum * 100);
+                        });
+                        // row.push(`${build.testruns[0].project.name}/${build.testruns[0].release.name}/${build.testruns[0].build.name}`);
+                        $scope.healthDataByProject[project].addRow(row);
+                    });
```

Comment:
:point_down: Only continue to query the Health check report if a `Health` release was initially found. Need to refresh if a `Health` release is added after the page is loaded.